common/src/main/java/org/wso2/testgrid/common/config/ScenarioConfig.java

```diff
@@ -163,7 +162,7 @@ public void setRemoteBranch(String remoteBranch) {
      *
      * @return List of inputParameters
      */
-    public Map<String, String> getInputParameters() {
+    public HashMap getInputParameters() {
```

Comment:
Using HashMap is an internal detail of this class. So, we should not expose that to outside of this class. So, I suggest we use the return type as Map. May be with generic types as Map<String, Object>.